dotcom-rendering/src/static/icons/pin.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="60" height="105" viewBox="0 0 60 105" fill="none" >
+<path d="M60 56.9C56.5 49.6 50.4 43.7 42.9 40.5L40.1 7.9C45.1 6.3 49.5 3.6 53.2 0H39.5H20.5H6.8C10.4 3.6 14.9 6.3 19.9 7.9L17.2 40.5C9.6 43.7 3.5 49.6 0 56.9H15.8H25.3V85.4L30 104.4L34.7 85.4V56.9H44.2H60Z" />
+</svg>
```

Comment:
Now https://github.com/guardian/source/pull/1292 is merged, can we remove this file?